ghost/core/core/frontend/services/theme-engine/middleware/update-local-template-options.js

```diff
@@ -9,7 +9,7 @@ function updateLocalTemplateOptions(req, res, next) {
 
     // adjust @site.url for http/https based on the incoming request
     const siteData = {
-        url: urlUtils.urlFor('home', {trailingSlash: false}, true)
```

Comment:
Could you test this setup with storages? I think storages uses this base url to construct uploaded file urls. It will be nice to be sure this doesn't break anything (I'm not sure if we have automated tests cover those cases).

Reply:
@muratcorlu thanks for reviewing. That's a good question. I think a Ghost backend developer could answer that question pretty quickly – certainly much quicker than me.

I think if storages are simply appending strings to the base URL, there should be a separate PR to make that more intelligent and bug-proof. If there is also missing test coverage, it should probably be added at the same time.